install_config/web_console_customization.adoc

```diff
@@ -174,6 +174,36 @@ assetConfig:
 ====
 endif::[]
 
+[[enable-tech-preview-feature]]
+=== Enabling Features in Tech Preview
+
+Sometimes features will be available in tech preview, by default these features may be hidden
+from end users.  
+
+The current list of features in tech preview is:
+
+* pipelines - Enabling this will add the Pipelines navigation item underneath the Builds menu
```

Comment:
@spadgett @jwforres Wait, is the reader supposed to replace (in this example ) "pipelines" where it says "feature_name" in the below example? If that's the case, then that definitely needs to be clarified.